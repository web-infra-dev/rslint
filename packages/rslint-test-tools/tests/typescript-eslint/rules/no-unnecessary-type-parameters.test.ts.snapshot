exports[`no-unnecessary-type-parameters > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-unnecessary-type-parameters",
      "messageId": "sole",
      "message": "Type parameter T is used only once in the function signature.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 15
        },
        "end": {
          "line": 1,
          "column": 16
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-unnecessary-type-parameters > invalid 2`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-unnecessary-type-parameters",
      "messageId": "sole",
      "message": "Type parameter T is used only once in the function signature.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 15
        },
        "end": {
          "line": 1,
          "column": 16
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-unnecessary-type-parameters > invalid 3`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 1
}
`;
