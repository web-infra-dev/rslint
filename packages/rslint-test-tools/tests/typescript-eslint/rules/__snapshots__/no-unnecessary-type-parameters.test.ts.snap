// Rstest Snapshot v1

exports[`no-unnecessary-type-parameters > invalid 1`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type parameter T is used only once in the function signature.",
      "messageId": "sole",
      "range": {
        "end": {
          "column": 16,
          "line": 1,
        },
        "start": {
          "column": 15,
          "line": 1,
        },
      },
      "ruleName": "no-unnecessary-type-parameters",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-unnecessary-type-parameters > invalid 2`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type parameter T is used only once in the function signature.",
      "messageId": "sole",
      "range": {
        "end": {
          "column": 16,
          "line": 1,
        },
        "start": {
          "column": 15,
          "line": 1,
        },
      },
      "ruleName": "no-unnecessary-type-parameters",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-unnecessary-type-parameters > invalid 3`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 1,
}
`;
